### PR TITLE
[skip ci] Added more info about logs

### PR DESCRIPTION
**Pull request by stuclem**
I want to merge this so that Nandita can access the latest versions.

### CODE REVIEW

**Comment by stuclem** [comment]
LGTM

**Comment by npakrasi** [comment]
LGTM

File: doc/user_doc/vic_admin/vicadmin_status_ref.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+The Web-based administration portal for virtual container hosts, VCH Admin, presents status information about a virtual container host.
+
+If the vSphere environment in which you are deploying a virtual container host does not meet the requirements, the deployment does not succeed. However, a successfully deployed virtual container host can stop functioning if the vSphere environment changes after the deployment. If environment changes adversely affect the virtual container host, the status of the affected component changes from green to yellow or red.
+
+## Virtual Container Host (VCH) ##
+
+VCH Admin checks the status of the processes that the virtual container host runs:
+
+- The port layer server, that ports Docker commands to vSphere API calls.
+- VIC Admin server, that
```
Comment (hickeng) [inline doc/user_doc/vic_admin/vicadmin_status_ref.md]: sentence fragment

File: doc/user_doc/vic_admin/vicadmin_status_ref.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+If the vSphere environment in which you are deploying a virtual container host does not meet the requirements, the deployment does not succeed. However, a successfully deployed virtual container host can stop functioning if the vSphere environment changes after the deployment. If environment changes adversely affect the virtual container host, the status of the affected component changes from green to yellow or red.
+
+## Virtual Container Host (VCH) ##
+
+VCH Admin checks the status of the processes that the virtual container host runs:
+
+- The port layer server, that ports Docker commands to vSphere API calls.
+- VIC Admin server, that 
+- The vSphere Integrated Containers Engine initialization service, that initializes virtual container hosts.
```
Comment (hickeng) [inline doc/user_doc/vic_admin/vicadmin_status_ref.md]: `Initialization and watchdog service for the other components`

File: doc/user_doc/vic_admin/vicadmin_status_ref.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+If the vSphere environment in which you are deploying a virtual container host does not meet the requirements, the deployment does not succeed. However, a successfully deployed virtual container host can stop functioning if the vSphere environment changes after the deployment. If environment changes adversely affect the virtual container host, the status of the affected component changes from green to yellow or red.
+
+## Virtual Container Host (VCH) ##
+
+VCH Admin checks the status of the processes that the virtual container host runs:
+
+- The port layer server, that ports Docker commands to vSphere API calls.
+- VIC Admin server, that 
+- The vSphere Integrated Containers Engine initialization service, that initializes virtual container hosts. 
+- The Docker engine server, that runs the vSphere Integrated Containers Engine implementations of Docker commands.
```
Comment (hickeng) [inline doc/user_doc/vic_admin/vicadmin_status_ref.md]: `implementations of the Docker API`

File: doc/user_doc/vic_admin/vicadmin_status_ref.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+VCH Admin checks that the firewall is correctly configured on the ESXi host or the ESXi hosts in the cluster on which the virtual container host is running.
+
+**Error message**
+
+The Firewall status is yellow and shows the error `Firewall must permit 2377/tcp outbound to use VIC`.
+
+**Cause**
+
+The firewall on the ESXi host on which the virtual container host is running no longer allows outbound connections on port 2377.
```
Comment (hickeng) [inline doc/user_doc/vic_admin/vicadmin_status_ref.md]: This and the license check can also fail if the `management` network is down, or if the endpointVM is unable to connect to vSphere.

File: doc/user_doc/vic_admin/vicadmin_status_ref.md (hunk 1 above)
Comment (stuclem) [inline doc/user_doc/vic_admin/vicadmin_status_ref.md]: Oops. Expanded to "VIC Admin server, that runs the VCH Admin portal. "

File: doc/user_doc/vic_admin/vicadmin_status_ref.md (hunk 2 above)
Comment (stuclem) [inline doc/user_doc/vic_admin/vicadmin_status_ref.md]: Updated to "The vSphere Integrated Containers Engine initialization service and watchdog service for the other components."

File: doc/user_doc/vic_admin/vicadmin_status_ref.md (hunk 3 above)
Comment (stuclem) [inline doc/user_doc/vic_admin/vicadmin_status_ref.md]: Expanded, per your version on Slack, to "The Docker engine server, that exposes the Docker API and semantics, translating those composite operations into port layer primitives."

**Pull request merged**